playbook/dist/playbook.css

```diff
@@ -2906,9 +2906,12 @@ Background colors ------------------*/
     [class^=pb_badge_kit].dark[class*=_primary] {
       background: rgba(1, 78, 190, 0.2); }
 
+<<<<<<< HEAD
```

Comment:
You've got merge conflicts here. You can reset this single file to master by running `git checkout origin/master playbook/dist/playbook.css` then push up that change

Reply:
will do